src/rustdoc-json-types/lib.rs

```diff
@@ -1036,8 +1036,11 @@ pub enum Type {
 /// A type that has a simple path to it. This is the kind of type of structs, unions, enums, etc.
 #[derive(Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize)]
 pub struct Path {
-    /// The name of the type.
-    pub name: String,
+    /// The path of the type.
+    ///
+    /// This will be the path where the type is *used*, so it may be a
+    /// re-export.
```

Comment:
Suggestion, feel free to take it or leave it: I'm not 100% sure what "used" means in this context. Is it the path referenced in the code being documented, either directly or via an import? AKA "what path would we have to remove to break the code being documented"

Perhaps an example could help?

Reply:
Added an example:

```rust
pub type Vec1 = std::vec::Vec<i32>; // path: "std::vec::Vec"
pub type Vec2 = Vec<i32>; // path: "Vec"
pub type Vec3 = std::prelude::v1::Vec<i32>; // path: "std::prelude::v1::Vec"
```

Reply:
Thank you!